### DISCUSSION TITLE
rAudio 1 • 20211112: Creating new webradios not possible

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#606] [Fixed Issues]
During tests of the new subfolder structure of webradios I made the experience that editing of existing webradios, e.g. new name or new address, or creating a new webradio is not possible. No error message but change is not present.

**Comment by klassikmann** [comment]
Output of journalctl of  a "create new webradio":

Nov 12 14:32:04 rAudio audit[12983]: USER_ACCT pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:accounting grantors=pam_unix,pam_permit,pam_time acct="http" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio kernel: audit: type=1101 audit(1636723924.179:764): pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:accounting grantors=pam_unix,pam_permit,pam_time acct="http" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio sudo[12983]:     http : PWD=/srv/http ; USER=root ; COMMAND=/srv/http/bash/cmd.sh webradioadd
                                    FFH
                                    https://mp3.ffh.de/radioffh/hqlivestream.mp3
                                    WEBRADIO

Nov 12 14:32:04 rAudio audit[12983]: CRED_REFR pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,pam_env,pam_faillock acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio audit[12983]: USER_START pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_open grantors=pam_limits,pam_unix,pam_permit acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio sudo[12983]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=33)

Nov 12 14:32:04 rAudio kernel: audit: type=1110 audit(1636723924.183:765): pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,pam_env,pam_faillock acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio kernel: audit: type=1105 audit(1636723924.183:766): pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_open grantors=pam_limits,pam_unix,pam_permit acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio sudo[12983]: pam_unix(sudo:session): session closed for user root

Nov 12 14:32:04 rAudio audit[12983]: USER_END pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_close grantors=pam_limits,pam_unix,pam_permit acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio audit[12983]: CRED_DISP pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,pam_env,pam_faillock acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio kernel: audit: type=1106 audit(1636723924.259:767): pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_close grantors=pam_limits,pam_unix,pam_permit acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

Nov 12 14:32:04 rAudio kernel: audit: type=1104 audit(1636723924.259:768): pid=12983 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,pam_env,pam_faillock acct="root" exe="/usr/bin/sudo" hostname=? addr=? terminal=? res=success'

**Reply by rern** [reply]
Try the UPDATE.

**Reply by klassikmann** [reply]
Perfect. Thank you.
